portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase

```diff
@@ -3731,6 +3731,689 @@ definition {
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection Test");
+
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "First Segment");
+
+			AssetListsAdmin.configureItemTypeInDynamicCollection(
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "First Segment");
+		}
+
+		task ("Create a second variation for the collection") {
+			AssetListsAdmin.addPersonalizedVariation(segmentName = "Second Segment");
+
+			AssetListsAdmin.configureItemTypeInDynamicCollection(
+				itemSubtype = "All Subtypes",
+				itemType = "Web Content Article");
+
+			AssetListsAdmin.prioritizeVariation(variationTitle = "Second Segment");
+		}
+
+		task ("Create a public page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Page Name",
+				type = "content");
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+		}
+
+		task ("Add a Collection Display to page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.addFragment(
+				collectionName = "Content Display",
+				fragmentName = "Collection Display");
+		}
+
+		task ("Select the manual collection in the Collection Display") {
+			PageEditor.editCollectionDisplay(
+				assetListName = "Dynamic Collection Test",
+				fragmentName = "Collection Display",
+				styleDisplay = "Bulleted List");
+
+			Button.clickPublish();
+
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/test-site-name/test-page-name");
+		}
+
+		task ("Assert the asset appear only once in Collection Display Fragment") {
+			AssertTextEquals(
+				index = "1",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_4");
+
+			AssertTextEquals(
+				index = "2",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_3");
+
+			AssertTextEquals(
+				index = "3",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_2");
+
+			AssertTextEquals(
+				index = "4",
+				locator1 = "Fragment#CONTENT_DISPLAY_ASSET_SUMMARY",
+				value1 = "WC_Title_1");
+		}
+
+		task ("Set collections configuration to default values") {
+			AssetListsAdmin.resetCollectionsToCombineAssetsFromSegments();
+		}
+	}
+
+	@description = "LPS-169312: Given a user belonging to Segment 1 and Segment 2
+	And the configuration to combine assets is enabled
+	When there's a variation of a manual collection targeting segment 1 and segment 2, and both are prioritized above the anyone variation
+	And there are some assets present in both variations**
+	Then the assets shown to the user will be ordered according to:
+	The priority of the Variation
+	The priority of the asset established in the Variation"
+	@priority = "3"
+	test ViewCombineContentOrderForMultisegmentManualCollection {
+		property portal.acceptance = "true";
```

Comment:
This test has priority 3 so it is not considered as acceptance. Remove this property.